ui/app/src/components/RecentActivityDashlet/RecentActivityDashlet.tsx

```diff
@@ -163,17 +163,14 @@ export default function RecentActivityDashlet() {
 
   // region Item Updates Propagation
   useEffect(() => {
-    const events = [itemsDeleted.type, itemsRejected.type, itemsApproved.type, itemsScheduled.type];
+    const events = [deleteContentEvent.type, workflowEvent.type, publishEvent.type]; // TODO: should contentEvent be here?
     const hostToHost$ = getHostToHostBus();
     const subscription = hostToHost$.pipe(filter((e) => events.includes(e.type))).subscribe(({ type, payload }) => {
       switch (type) {
-        case itemsApproved.type:
-        case itemsScheduled.type:
-        case itemsDeleted.type:
-        case itemsRejected.type: {
-          if (payload.targets.some((path) => selectedLookup[path])) {
-            fetchActivity();
-          }
+        case deleteContentEvent.type:
+        case workflowEvent.type:
+        case publishEvent.type: {
+          fetchActivity();
```

Comment:
The whole switch is kind of pointless as the handling is same for all listened events.